src/deviation.jl

```diff
@@ -51,7 +82,14 @@ function L1dist{T<:Number}(a::AbstractArray{T}, b::AbstractArray{T})
     return r
 end
 
+
 # Linf distance
+"""
+    Linfdist(a, b)
+
+Compute the L∞ distance, also called the Chebyshev distance, between
+two arrays: `max|a-b|`.
```

Comment:
By the way, I know double backticks can be used for LaTex notations, you might want to try that.